sql/core/src/main/scala/org/apache/spark/sql/internal/BaseSessionStateBuilder.scala

```diff
@@ -146,6 +147,15 @@ abstract class BaseSessionStateBuilder(
     extensions.buildParser(session, new SparkSqlParser())
   }
 
+  /**
+   * Script interpreter that produces execution plan for sql batch procedural language.
```

Comment:
maybe change this message to something cleaner, like "Interpreter that produces execution plan and executes SQL scripts.